core/src/components/picker-column/picker-column.tsx

```diff
@@ -206,6 +215,10 @@ export class PickerColumn implements ComponentInterface {
     }
   }
 
+  connectedCallback() {
+    this.ariaLabel = this.el.getAttribute('aria-label') ?? 'Select a value';
```

Comment:
I could be convinced to not have a default value here since the label isn't super helpful. However, I was concerned about breaking the existing experience since not using an `aria-label` on the slider role could cause tools such as Axe to start failing.

Reply:
I think a label is better than no label here. It is also incredibly straightforward for developers to customize it.